acceptance/tests/pxp_agent_version.rb

```diff
@@ -1,12 +1,17 @@
 test_name 'C93805 - pxp-agent - Versioning test'
 
-agent1 = agents[0]
+def pxp_version_cmd(platform)
+  if platform.start_with?('windows')
+    # simple hack to ensure OpenSSL is found w/out modifying PATH
+    "cd /cygdrive/c/Program\\ Files/Puppet\\ Labs/Puppet/sys/ruby/bin && ../../../pxp-agent/bin/pxp-agent --version"
+  else
+    'cd /opt/puppetlabs/puppet/bin && ./pxp-agent --version'
+  end
```

Comment:
Puppet acceptance has a helper for executing other private commands: https://github.com/puppetlabs/puppet/blob/master/acceptance/lib/puppet/acceptance/common_utils.rb#L128-L147. Since privatebindir on Windows includes both "Program Files" and "Program Files (x86)", it works for both 64 and 32-bit puppet.